default/scripting/buildings/GATEWAY_VOID.focs.txt

```diff
@@ -27,19 +27,20 @@ BuildingType
                     ]
                     empire = Source.Owner
             ]
+            
```

Comment:
Nitpick: many of the added "blank" lines in .focs.txt files contain excess whitespace